ACE/ace/Svc_Conf_Lexer.cpp

```diff
@@ -34,7 +34,7 @@ ACE_BEGIN_VERSIONED_NAMESPACE_DECL
 # define ACE_TEMPORARY_STRING(X,SIZE) \
    char* X = 0; \
    char X ## buf[ACE_YY_BUF_SIZE]; \
-   std::unique_ptr<char> X ## bufp (0); \
```

Comment:
Note: this make problems with wchar_t under windows!

Reply:
Probably can be solved by removing `(0)`

Reply:
sorry, no of my time left. 
This should be part of the next step ...